doc(release): Release notes for OSS 23.10 (#3446)
centreon/centreon-documentation: versioned_docs/version-23.10/releases/centreon-os.mdx

## Centreon Web

### 23.10.14

Release date: `July 18, 2024`

<details>
  <summary>Bug fixes</summary>

> - [Gorgone] Fixed an issue where a host category was not present in MBI if it was applied on a host template instead of directly on the host.
> - [Menu] Updated url of helper button in web interface.
> - [Unattended] Backported recent bug fixes on unattended (http services not restarting, it-edition-extensions installation, unattended issue on Oracle Linux distributions).

Reviewer: ```suggestion
- [Menu] Updated the URLs of help buttons in the web interface.
- [Unattended] Backported recent bug fixes on the Unattended script (http services not restarting, it-edition-extensions installation, unattended issue on Oracle Linux distributions).
```
